src/main/scala/com/gu/support/workers/model/ErrorState.scala

```diff
@@ -0,0 +1,3 @@
+package com.gu.support.workers.model
+
+case class ErrorState(Error: String, Cause: String = "")
```

Comment:
Didn't realise it was implementing an AWS model.  I think it is fine for it to stay title-case.